### PR TITLE
add plugin for draft

### PR DESCRIPTION
**Pull request by tomdavidson**
Adds plugin for Azure Draft

### CODE REVIEW

**Comment by Stratus3D** [comment]
It looks like you've got a failing build on OSX.

**Comment by lalloni** [comment]
Works for me.

File: plugins/draft [hunk 1]
```diff
@@ -0,0 +1 @@
+repository = https://gitlab.com/td7x/asdf/draft/
```
Comment (Stratus3D) [inline plugins/draft]: This should be the Git url: https://gitlab.com/td7x/asdf/draft.git

**Comment by Stratus3D** [comment]
@tomdavidson can you rebase this PR onto the latest master? Many things have changed.

File: plugins/draft (hunk 1 above)
Comment (imbsky) [inline plugins/draft]:
```suggestion
repository = https://gitlab.com/td7x/asdf/draft.git
```

**Comment by imbsky** [comment]
Draft plugin already exists in this repository, so I will close this PR.

**Pull request closed without merging**